programmers/짝수와-홀수.js

```diff
@@ -0,0 +1,10 @@
+function solution(num) {
+    var answer =''
+    if(num%2 == 0){
+        answer = "Even";
+    }
+    else{
+        answer = "Odd";
+    }
```

Comment:
```suggestion
        answer = "Even";
    } else {
        answer = "Odd";
    }
```
이건 규칙은 아니지만 이렇게 많이 사용합니다.